Formula/v.rb

```diff
@@ -3,6 +3,7 @@ class V < Formula
   homepage "https://github.com/rupa/v"
   url "https://github.com/rupa/v/archive/v1.1.tar.gz"
   sha256 "6483ef1248dcbc6f360b0cdeb9f9c11879815bd18b0c4f053a18ddd56a69b81f"
+  revision 1
```

Comment:
Ah, I think confusion was that it failed during test because `vim` was broken.  Had I run a test where `vim` was revision bumped but `v` was not, it would have succeeded. We'll be able to skip revision bumping this when we upgrade to Perl 5.36.

Reply:
Well, that's why we rev bump linkage failures before touching any test failures.